### PR TITLE
Fix media browser gender translations

### LINKED ISSUE #14526: Gender translations not loaded for cloud TTS
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Opening media browser from Lovelace and navigation to cloud TTS will lack the translation of the gender selector

### Describe the behavior you expected

It should be translated 

### Steps to reproduce the issue

1. reopen app
2. Click media browser icon on media player 
3. Navigate to cloud TTS
...

### What version of Home Assistant Core has the issue?

2022.11

### What was the last working version of Home Assistant Core?

_No response_

### In which browser are you experiencing the issue with?

All

### Which operating system are you using to run this browser?

Mac

### State of relevant entities

_No response_

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by piitaya**
## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #14526
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/data/cloud/tts.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const getCloudTtsSupportedGenders = (
       genders.push([
         gender,
         gender === "male" || gender === "female"
-          ? localize(`ui.panel.config.cloud.account.tts.${gender}`)
```
Comment (steverep) [inline src/data/cloud/tts.ts:63]: If this key didn't exist, TypeScript would throw an error.  Is the key wrong?

Comment (bramkragten) [reply src/data/cloud/tts.ts:63]: No, but the config translations will not be loaded in the media browser, they are only loaded on the config panel

**Review by bramkragten — DISMISSED** [review]

File: src/data/cloud/tts.ts (hunk 1 above)
Comment (bramkragten) [reply src/data/cloud/tts.ts:63]: Should they be removed from the translations now? Or are they used somewhere else still?

Comment (steverep) [reply src/data/cloud/tts.ts:63]: Ohhh... if I remember correctly, this was the only spot so they can probably be removed.

Comment (piitaya) [reply src/data/cloud/tts.ts:63]: Yes, I will remove them 🙂

**Review by bramkragten — APPROVED** [review]

**Pull request merged**